### DISCUSSION TITLE
Fraud Example

### DISCUSSION THREAD

**Discussion by igorwiese** [dagster-io/dagster#13790] [Q&A]
Hello Everyone. I was reading the Dagster Blog explaining the ML pipelines (https://dagster.io/blog/dagster-ml-pipelines). I loved the tool and the article. Is this fraud example presented in the article available?

Best,
Igor Wiese

**Comment by tacastillo** [comment]
Hi Igor!

Glad that you liked the project.

I did a quick search of the private and public Dagster repos and couldn't find it.

I think the next step would be to ask the origin of the code, @sryza, if he has it pushed up somewhere.

In the meanwhile, here is a [similar complex project](https://github.com/dagster-io/hooli-data-eng-pipelines) that might demonstrate the same concepts you liked in the fraud one.
